regress/loader/TestSkipANALYZE.select.sql

```diff
@@ -0,0 +1,5 @@
+-- Checks if the table pgreg.loadedshp was analyzed by
```

Comment:
Maybe add a similar check in another test NOT using -Z to verify ANALYZE *is* called ?

Reply:
Yes, this is actually a very good idea and make the test suite more complete. Just added a negative test case to check what happens in the absence of -Z.